package.json

```diff
@@ -96,5 +95,8 @@
     "compilerOptions": {
       "esModuleInterop": true
     }
+  },
+  "dependencies": {
+    "dicer": "^0.3.0"
```

Comment:
I'm -1 on using dicer.

Reply:
![image](https://user-images.githubusercontent.com/36894700/129530646-67618e7a-38b5-4e2f-bcc1-713e8b234e5a.png)

Reply:
It works... what would you prefer? I'd like to replace it with our own implementation. However, I do not have time to do that atm. Can we land it with dicer as experimental and then make replacing it a requirement for stable release?

Reply:
Also there are more question with `FormData` that we need to sort out https://github.com/nodejs/undici/issues/964

Reply:
fastify-multipart uses busboy which uses dicer

Reply:
It's the best option I can find on npm.

Reply:
oh I know that it's the best on npm. I'm ok in using dicer for now, but I would also like to keep the "no dependency" approach.

We might consider vendoring it and iterating.

Reply:
Does that mean you are fine with landing this in it's current form for now?